1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,23 @@
 // Write your code here
+// Write your code here
```

Comment:
Really nice stuff again Saf!!! I love the styling and that it is center aligned, this makes me very happy!!! 

It took my months to really understand `setInterval`, but you clearly are quickily becoming a pro! Great work! 🎉  🙌  🕺